deployments/optimism/usdc/configuration.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "Compound USDC",
+  "symbol": "cUSDCv3",
+  "baseToken": "USDC",
+  "baseTokenAddress": "0x0b2C639c533813f4Aa9D7837CAf62653d097Ff85",
+  "baseTokenPriceFeed": "0x16a9FA2FDa030272Ce99B29CF780dFA30361E0f3",
+  "pauseGuardian": "0x3fFd6c073a4ba24a113B18C8F373569640916A45",
+  "borrowMin": "1e0",
+  "storeFrontPriceFactor": 0.6,
+  "targetReserves": "20000000e6",
+  "rates": {
+    "supplyKink": 0.9,
+    "supplySlopeLow": 0.059,
+    "supplySlopeHigh": 2.9,
+    "supplyBase": 0,
+    "borrowKink": 0.9,
+    "borrowSlopeLow": 0.061,
+    "borrowSlopeHigh": 3.2,
+    "borrowBase": 0.015
+  },
+  "tracking": {
+    "indexScale": "1e15",
+    "baseSupplySpeed": "57_870_370_370e0",
+    "baseBorrowSpeed": "57_870_370_370e0",
+    "baseMinForRewards": "1000e6"
+  },
+  "assets": {
+    "WETH": {
+      "address": "0x4200000000000000000000000000000000000006",
+      "priceFeed": "0x13e3Ee699D1909E989722E753853AE30b17e08c5",
+      "decimals": "18",
+      "borrowCF": 0.83,
+      "liquidateCF": 0.9,
+      "liquidationFactor": 0.95,
+      "supplyCap": "1600e18"
```

Comment:
Supply caps should also be 0 in the initial deployment

Reply:
Updated